packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -147,14 +161,21 @@ export default {
             type: String,
             default: ''
         },
+
         ariaLabel: {
             type: String,
             default: ''
         },
+
         titleHtmlTag: {
             type: String,
             default: 'h3',
             validator: value => ['h1', 'h2', 'h3', 'h4'].includes(value)
+        },
+
+        isModeRightToLeft: {
```

Comment:
Should this be prop-driven or just dependent on the writing-mode/directionality of the consuming application?

In other words, would we expect a consumer that supports multiple languages to pass in a prop that changes depending on the currently selected language?

Reply:
Yes so the consumer would be responsible for passing in a prop / boolean based on the tenant. Not sure if that answers the question?